statuspage/src/App.vue

```diff
@@ -57,14 +54,14 @@ import Vuex from "vuex";
 import store from "./store";
 import AutoUpdateSettings from "./components/AutoUpdateSettings.vue";
 import ErrorPanel from "./components/ErrorPanel.vue";
-import StatusStrip from "./components/StatusStrip.vue";
+import TableRow from "@/components/TableRow";
 
 export default {
   name: "App",
   components: {
+    TableRow,
```

Comment:
Maybe `CanaryStandardServerRow`? `TableRow` feels like it covers too much territory. Otherwise, nice work on this, you're picking up Vue quickly. :)

Reply:
Thanks and changed :)